packages/data/src/world/mq/dodongo_cavern_mq.yml

```diff
@@ -93,7 +93,7 @@
   dungeon: DC
   exits:
     "Dodongo Cavern Main Room Upper Entrance Side": "true"
-    "Dodongo Cavern Room Before Upper Lizalfos": "can_use_sticks || has_fire"
+    "Dodongo Cavern Room Before Upper Lizalfos": "(can_use_sticks && event(OPEN_MQ_DC_LARVAE_ROOM)) || has_fire"
```

Comment:
This can actually just be true right now, since the transition occurs before the web.

Reply:
is this wrong though? if its right, I'd rather just leave it

Reply:
It's fine. Just would need changed if plane transitions are ever shuffled.